CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+<h1 align="center">
+  <img src="docs/img/scala-tron.png">
+  <br>
+</h1>
+
+# Contributing
+scala-tron is an open source project.
+
+It is the work of contributors. We appreciate your help!
+
+Here are instructions to get you started. They are not perfect, so please let us know if anything feels wrong or incomplete.
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or on [Gitter](https://gitter.im/tronprotocol/java-tron) with the owners of this repository before making a change. 
```

Comment:
Telegram is also used as communication tool, you could add that here too. This is the group chat https://t.me/joinchat/CP8XKBIjEc0RqzJdl61OlQ

Reply:
The gitter link is that of java-tron